phpstan.neon

```diff
@@ -1,5 +1,5 @@
 parameters:
-    level: 6
+    level: 5
```

Comment:
@bennothommo how would you feel about this?

Reply:
Is level 6 where you start having to go into "generics" level detail? If so, I'm happy for you to drop it down.

Reply:
Can we keep it at 6 but ignore the specific rule that is causing issues or does it not really work like that?